tests/test_infographic_api.py

```diff
@@ -75,6 +79,57 @@ def test_bad_news_flash_id(self, app):
 
         assert rv.status_code == http_client.NOT_FOUND
 
+    def test_accident_count_by_car_type(self, app):
+        test_involved_by_vehicle_type_data = [{"involve_vehicle_type": 1, "count": 3}, {"involve_vehicle_type": 25, "count": 2},
+         {"involve_vehicle_type": 15, "count": 1}]
+        output_tmp = percentage_accidents_by_car_type(test_involved_by_vehicle_type_data)
+        assert len(output_tmp) == 3
+        assert output_tmp["רכב פרטי"] == 50
+        assert output_tmp["מסחרי/משאית"] == 33.333333333333336
+        assert output_tmp["אופניים/קורקינט"] == 16.666666666666668
```

Comment:
sorry, one last thing.. consider using  self.assertAlmostEqual()
thanks for adding the tests!

Reply:
assertAlmostEqual is python default unittest library function, we are using pytest - the best similarity I found is pytest.approx